Gopkg.lock

```diff
@@ -441,6 +441,25 @@
   pruneopts = "UT"
   revision = "e80d13ce29ede4452c43dea11e79b9bc8a15b478"
 
+[[projects]]
```

Comment:
I assume there should be a corresponding change to Gopkg.toml that pulls in the new deps?